sidechain/consensus/aura/src/slot_proposer.rs

```diff
@@ -68,7 +68,7 @@ where
 				latest_onchain_header,
 				executed_operation_hashes,
 				self.shard,
-				batch_execution_result.previous_state_hash,
+				batch_execution_result.initial_state_hash,
```

Comment:
I am not sure, if I agree 100% with the renaming here. What was your thinking?

Reply:
previous indicates that there have been changes and there is a new one somewhere. But if there are no actual changes it didn't fit right.

Reply:
Ah, I see. You are right that this is suboptimal, but I think it is neither the `initial` (if I am not mistaken), but rather the `current`.

Reply:
But current does not fit in case there are changes? Unsure if that will be the case with the current set up

Reply:
Which is why it was previous before. :D

So instead of previous, we could name it `apriori` as elsewhere, or probably better `state_hash_before_execution`

Reply:
Added the link to this discussion to https://github.com/integritee-network/worker/pull/520 as a TODO. I'll think about that once I've applied the state mutation changes. Maybe that will clarify when state will be changed and when not.